images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -4,7 +4,9 @@ function Get-AnsibleVersion {
 }
 
 function Get-AptFastVersion {
-    $aptFastVersion = (dpkg-query --showformat='${Version}' --show apt-fast).Split('-')[0]
+    $versionFileContent = Get-Content "/usr/bin/apt-fast" -Raw
```

Comment:
apt-fast is installed into `/usr/local/sbin/apt-fast` when using the installation script

Reply:
Let's do `Get-Content (which apt-fast) -Raw`